src/bundle.c

```diff
@@ -556,6 +581,7 @@ gboolean umount_bundle(RaucBundle *bundle, GError **error) {
 void free_bundle(RaucBundle *bundle) {
 
 	g_free(bundle->path);
+	g_clear_pointer(&bundle->mount_point, g_free);
```

Comment:
No need to use `g_clear_pointer()` here, instead directly use `g_free()` which handles NULL pointers.

Reply:
You are right. I've pushed a fix.